<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>236</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Template:</string>
       </property>
       <property name="buddy">
        <cstring>comboTemplate</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboTemplate"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Class:</string>
       </property>
       <property name="buddy">
        <cstring>editClass</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editClass"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Version:</string>
       </property>
       <property name="buddy">
        <cstring>editVersion</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editVersion"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Author:</string>
       </property>
       <property name="buddy">
        <cstring>editAuthor</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editAuthor"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Email:</string>
       </property>
       <property name="buddy">
        <cstring>editEmail</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editEmail"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Copyright:</string>
       </property>
       <property name="buddy">
        <cstring>editCopyright</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editCopyright"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelBaseCalss">
       <property name="text">
        <string>Base class:</string>
       </property>
       <property name="buddy">
        <cstring>editBaseClass</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editBaseClass"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonGenerate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboTemplate</tabstop>
  <tabstop>editClass</tabstop>
  <tabstop>editBaseClass</tabstop>
  <tabstop>editVersion</tabstop>
  <tabstop>editAuthor</tabstop>
  <tabstop>editEmail</tabstop>
  <tabstop>editCopyright</tabstop>
  <tabstop>buttonGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
